cloudigrade/account/tests/helper.py

```diff
@@ -119,3 +122,33 @@ def generate_aws_instance_events(
             )
             events.append(event)
     return events
+
+
+def generate_aws_image(account,
+                       ec2_ami_id=None,
+                       is_encrypted=False,
+                       is_windows=False):
+    """
+    Generate an AwsMachineImage for the AwsAccount for testing.
+
+    Any optional arguments not provided will be randomly generated.
+
+    Args:
+        account (AwsAccount): Account that owns the instance.
+        ec2_ami_id (str): Optional EC2 ami id.
+        is_encrypted (bool): Optional Indicates if image is encrypted.
+        is_windows (bool): Optional Indicates if AMI is Windows.
+
+    Returns:
+        AwsMachineImage: The created AwsMachineImage.
+
+    """
+    if ec2_ami_id is None:
```

Comment:
codecov indicates that this conditional is _always_ executed and never skipped (and reading through the test code appears to confirms this). Since we never actually pass a value for `ec2_ami_id` info this function, we should just remove the parameter and drop the condition so the value is always generated by the helper.

Reply:
👍